--- lxqt-panel-0.10.0/panel/resources/panel.conf.0000~	2015-11-13 15:50:12.354868659 +0100
+++ lxqt-panel-0.10.0/panel/resources/panel.conf	2015-11-13 15:53:38.249860897 +0100
@@ -2,25 +2,32 @@ panels=panel1
 
 [panel1]
 plugins=mainmenu,desktopswitch,quicklaunch,taskbar,tray,statusnotifier,mount,volume,clock,showdesktop
+iconSize=32
+panelSize=32
 position=Bottom
 desktop=0
 
 [mainmenu]
+alignment=left
 type=mainmenu
+text=OpenMandriva
+ShowText=true
 
 [desktopswitch]
 type=desktopswitch
 
 [quicklaunch]
 type=quicklaunch
-apps/size=5
-apps/1/desktop=firefox.desktop
-apps/2/desktop=chromium-browser.desktop
-apps/3/desktop=pcmanfm-qt.desktop
-apps/5/desktop=lxqt-config.desktop
+apps/size=4
+apps/1/desktop=/usr/share/applications/qupzilla.desktop
+apps/2/desktop=/usr/share/applications/pcmanfm-qt.desktop
+apps/3/desktop=/usr/share/applications/qterminal.desktop
+apps/4/desktop=/usr/share/applications/openmandriva-drakconf.desktop
+alignment=Left
 
 [taskbar]
 type=taskbar
+alignment=Right
 
 [tray]
 type=tray
@@ -30,10 +37,15 @@ type=mount
 
 [clock]
 type=clock
+autoRotate=true
+showDate=below
+dateFormat=yyyy-MM-dd
+timeFormat=HH:mm
 
 [volume]
 device=0
 type=volume
+alignment=Right
 
 [showdesktop]
 alignment=Right
