diff -Naur lxqt-panel-0.10.0/panel/resources/panel.conf lxqt-panel-0.10.0.tpg/panel/resources/panel.conf
--- lxqt-panel-0.10.0/panel/resources/panel.conf	2015-10-31 00:22:47.000000000 +0000
+++ lxqt-panel-0.10.0.tpg/panel/resources/panel.conf	2016-02-03 19:51:49.530324957 +0000
@@ -2,25 +2,48 @@
 
 [panel1]
 plugins=mainmenu,desktopswitch,quicklaunch,taskbar,tray,statusnotifier,mount,volume,clock,showdesktop
+width=100
+width-percent=true
+iconSize=32
+panelSize=38
 position=Bottom
 desktop=0
 
 [mainmenu]
+alignment=left
 type=mainmenu
+text=OpenMandriva
+showText=true
+menu_file=/etc/xdg/menus/lxqt-applications.menu
 
 [desktopswitch]
 type=desktopswitch
+rows=2
 
 [quicklaunch]
 type=quicklaunch
-apps/size=5
-apps/1/desktop=firefox.desktop
-apps/2/desktop=chromium-browser.desktop
-apps/3/desktop=pcmanfm-qt.desktop
-apps/5/desktop=lxqt-config.desktop
+apps/size=4
+apps/1/desktop=/usr/share/applications/qupzilla.desktop
+apps/2/desktop=/usr/share/applications/pcmanfm-qt.desktop
+apps/3/desktop=/usr/share/applications/qterminal.desktop
+apps/4/desktop=/usr/share/applications/openmandriva-drakconf.desktop
+alignment=Left
 
 [taskbar]
 type=taskbar
+alignment=Right
+autoRotate=true
+buttonHeight=100
+buttonStyle=IconText
+buttonWidth=400
+closeOnMiddleClick=true
+groupingEnabled=true
+raiseOnCurrentDesktop=false
+showDesktopNum=0
+showGroupOnHover=true
+showOnlyCurrentScreenTasks=false
+showOnlyMinimizedTasks=true
+showOnlyOneDesktopTasks=false
 
 [tray]
 type=tray
@@ -30,10 +53,16 @@
 
 [clock]
 type=clock
+alignment=Right
+autoRotate=true
+showDate=below
+dateFormat=yyyy-MM-dd
+timeFormat=HH:mm
 
 [volume]
 device=0
 type=volume
+alignment=Right
 
 [showdesktop]
 alignment=Right
