diff -up lxqt-panel-0.13.0/panel/resources/panel.conf.0000~ lxqt-panel-0.13.0/panel/resources/panel.conf
--- lxqt-panel-0.13.0/panel/resources/panel.conf.0000~	2018-05-21 21:24:34.000000000 +0200
+++ lxqt-panel-0.13.0/panel/resources/panel.conf	2018-07-04 21:35:04.821943452 +0200
@@ -2,21 +2,48 @@ panels=panel1
 
 [panel1]
 plugins=mainmenu,desktopswitch,quicklaunch,taskbar,tray,statusnotifier,mount,volume,worldclock,showdesktop
+width=100
+width-precent=true
+iconSize=32
+panelSize=48
 position=Bottom
 desktop=0
 
 [mainmenu]
+alignment=left
 type=mainmenu
+text=OpenMandriva
+showText=true
+menu_file=/etc/xdg/menus/lxqt-applications.menu
 
 [desktopswitch]
 type=desktopswitch
+rows=2
 
 [quicklaunch]
 type=quicklaunch
+apps/size=4
+apps/1/desktop=/usr/share/applications/falkon.desktop
+apps/2/desktop=/usr/share/applications/pcmanfm-qt.desktop
+apps/3/desktop=/usr/share/applications/qterminal.desktop
+apps/4/desktop=/usr/share/applications/systemsettings.desktop
 alignment=Left
 
 [taskbar]
 type=taskbar
+alignment=Right
+autoRotate=true
+buttonHeight=100
+buttonStyle=IconText
+buttonWidth=400
+closeOnMiddleClick=true
+groupingEnabled=true
+raiseOnCurrentDesktop=false
+showDesktopNum=0
+showGroupOnHover=true
+showOnlyCurrentScreenTasks=false
+showOnlyMinimizedTasks=true
+showOnlyOneDesktopTasks=false
 
 [tray]
 type=tray
@@ -26,10 +53,16 @@ type=mount
 
 [worldclock]
 type=worldclock
+alignment=Right
+autoRotate=true
+showDate=below
+dateFormat=yyyy-MM-dd
+timeFormat=HH:mm
 
 [volume]
 device=0
 type=volume
+alignment=Right
 
 [showdesktop]
 alignment=Right
