diff -up lxqt-panel-1.1.0/panel/resources/panel.conf.1~ lxqt-panel-1.1.0/panel/resources/panel.conf
--- lxqt-panel-1.1.0/panel/resources/panel.conf.1~	2022-04-16 15:08:11.000000000 +0200
+++ lxqt-panel-1.1.0/panel/resources/panel.conf	2022-04-23 20:17:39.423317275 +0200
@@ -1,6 +1,10 @@
 panels=panel1
 
 [panel1]
+width=100
+width-precent=true
+iconSize=32
+panelSize=48
 plugins=mainmenu,desktopswitch,quicklaunch,taskbar,statusnotifier,tray,mount,volume,worldclock,showdesktop
 position=Bottom
 desktop=0
@@ -8,29 +12,55 @@ desktop=0
 [mainmenu]
 type=mainmenu
 filterClear=true
+alignment=left
+text=OpenMandriva
+showText=true
+menu_file=/etc/xdg/menus/lxqt-applications.menu
 
 [desktopswitch]
 type=desktopswitch
+rows=2
 
 [quicklaunch]
 type=quicklaunch
 alignment=Left
+apps/size=4
+apps/1/desktop=/usr/share/applications/falkon.desktop
+apps/2/desktop=/usr/share/applications/pcmanfm-qt.desktop
+apps/3/desktop=/usr/share/applications/qterminal.desktop
+apps/4/desktop=/usr/share/applications/systemsettings.desktop
 
 [taskbar]
 type=taskbar
-buttonWidth=220
+alignment=Right
+autoRotate=true
+buttonHeight=100
+buttonStyle=IconText
+buttonWidth=400
 closeOnMiddleClick=true
-groupingEnabled=false
+groupingEnabled=true
+raiseOnCurrentDesktop=false
+showDesktopNum=0
+showGroupOnHover=true
+showOnlyCurrentScreenTasks=false
+showOnlyMinimizedTasks=false
+showOnlyOneDesktopTasks=false
 
 [mount]
 type=mount
 
 [worldclock]
 type=worldclock
+alignment=Right
+autoRotate=true
+showDate=below
+dateFormat=yyyy-MM-dd
+timeFormat=HH:mm
 
 [volume]
 device=0
 type=volume
+alignment=Right
 
 [showdesktop]
 alignment=Right
